forc/src/ops/forc_init.rs

```diff
@@ -52,6 +53,40 @@ struct ContentResponse {
     url: String,
 }
 
+fn print_welcome_message() {
+    let read_the_docs = format!(
+        "Read the Docs:\n- {}\n- {}\n- {}",
+        Link::new("Sway Book", "https://fuellabs.github.io/sway/latest/"),
+        Link::new(
+            "Rust SDK Book",
+            "https://fuellabs.github.io/fuels-rs/latest/index.html"
+        ),
+        Link::new(
+            "Typescript SDK Docs",
+            "https://github.com/FuelLabs/fuels-ts"
+        )
+    );
+
+    let join_the_community = format!(
+        "Join the Community:\n- Follow us {}
+- Ask questions in dev-chat on {}",
+        Link::new("@SwayLang", "https://twitter.com/SwayLang"),
+        Link::new("Discord", "https://discord.com/invite/xfpK4Pe")
+    );
+
+    let report_bugs = format!(
+        "Report Bugs:\n- {}",
+        Link::new("Sway Issues", "https://github.com/FuelLabs/sway/issues/new")
+    );
+
+    let try_forc = "Now try `forc build` or `forc test`".to_string();
```

Comment:
Rephrase to something like: "to compile, run `forc build`, and to run tests run `forc test`"

Reply:
Does this need to be a String, since we will always know it's size maybe just leave it as a &str type instead?